src/shared/tests.c

```diff
@@ -360,3 +361,14 @@ const char *ci_environment(void) {
 
         return (ans = NULL);
 }
+
+int machine_id_initialized(void) {
+        sd_id128_t id;
+        int r;
+
+        r = id128_read("/etc/machine-id", ID128_FORMAT_PLAIN, &id);
```

Comment:
Why not simply use `sd_id128_get_machine()`?

Reply:
See #25734.

Reply:
Oh good point. I originally wrote this patch before your recent changes, and I wanted `ID128_PLAIN_OR_UNINIT`. I'll fix this.

Reply:
After #25734, you can simply write the check by `sd_id128_get_machine(NULL) >= 0`.

Reply:
Thanks, I rebased on your branch and will re-push once that is merged.